proposals/NNNN-HID-Support-Plug-in.md

```diff
@@ -0,0 +1,43 @@
+And, it's not realstic to add new RPC to support every OEM special device (exiting device and future device) because SDL core version management and Application vendor effort to support every spec. This proposal is "plugin architecture" to solve "OEM's specific device" without deep discussion to Commonization and Generalization of HID. Each OEM can provide their special device support module quickly with using this architecture. And, it's unnecessary Application vendors to customize for each OEM's specific device. On the other words, Application vendors can avoid customization for each OEM's specific device not only in pre-defined layout (as template) but also in video streaming.
+
+##Proposed solution
+
+Premise condition: "High level UI/Widget library" which support Android/iOS widget for rendering video streaming application in SDL proxy. ex. Android:Virtual Display Encoder, iPhone:XXXX This library can retrieve High level UI/Widget layout information and create High level UI/Widget event.
+This proposal architecture: Adding "Remote Human Interface Device manager" to SDL proxy (Android/iPhone). This module will provide following functionalities.
+dynamic selection of proper plugin for connected HU. This functionalitiy will be implemented with using VehicleType and reflection of Android Java and iOS ObjectiveC/Swift.
+bridge from/to "High-level UI/Widget library" to/from "plug-in module" which is proxy of HU special device.
+Adding "Plug-in for each OEM's specific device" which will be provided by each OEMs. This module may realize following functionalities.
+converting low level event which is sent from HU driver as OnTouchEvent(existing RPC) and/or OnSytemRequest(existing RPC, if OnTouchEvent's information is not enough to realize special event), to High level UI/Widget event, if required.
+sharing High level UI/Widget layout information with HU driver with using OnSystemRequest and/or other RPCs (existing RPC). This information may be used for haptic feedback and/or focus management in HU.
+superimposing information on application rendering image. (ex. cursor, focus highlight). These features requirements depend on each specific device features and OEM's UI/UX policy. NOTE) These feature may not be implemented in plugin. NOTE) Some features may be implemented in HU. NOTE) In some case of OEM specific plug-in, existing RPC may be used for not original purpose. However, this is closed spec between specific device and specific plug-in. So, it's not be affect to SDL general spec.
```

Comment:
Please reword second and third sentences to "These feature requirements depend on each specific device's features and OEM's UI/UX policy. NOTE: These features may not be implemented in plugin."

Also, please change the last sentence to "So, this will not affect the SDL general spec."